test/sm_test/sm_test.go

```diff
@@ -191,6 +192,17 @@ var _ = Describe("SM", func() {
 			BeforeEach(func() {
 				ctx = common.NewTestContextBuilder().Build()
 				smPlatformID = verifySMPlatformExists()
+				var updatedPlatform types.Object
+				byID := query.ByField(query.EqualsOperator, "name", types.SMPlatform)
+				platformFromStorage, err := ctx.SMRepository.Get(context.Background(), types.PlatformType, byID)
+				Expect(err).ToNot(HaveOccurred())
+
+				platformFromStorage.(*types.Platform).Active = false
+				if updatedPlatform, err = ctx.SMRepository.Update(context.Background(), platformFromStorage, types.LabelChanges{}); err != nil {
+					Expect(err).ToNot(HaveOccurred())
+				}
+				Expect(updatedPlatform.(*types.Platform).Active).To(Equal(false))
```

Comment:
why do we need to set SM platform to inactive?

Reply:
Removed (once we had a filter that handled delete objects flow, and that filter prevented tests in this case to work)